docs/guide/setup-examples/julia.md

```diff
@@ -0,0 +1,401 @@
+---
+title: How to use Sublime for Julia development
+---
+
+Adapted from https://github.com/PetrKryslUCSD/HowToUseJuliaWithSublimeText3
+This guide was written by a Windows user,
+that uses the Windows Subsystem for Linux to run Julia.
+
+When working with Julia,
+I like having my source code on the left
+and a terminal running Julia REPL on the right.
+This setup will allow you to have the terminal inside Sublime
+and to have commands that send code under your cursor to the REPL.
+
+## Installation
+
+### Install Julia syntax highlighting
+
+Install the [Julia][] package.
+Bring up the **Command Palette**,
+and type in a few letters of **Package Control: Install Package**.
+When it comes up, select it.
+Loading the registry will take a couple of seconds,
+and then we get window where we can type in `Julia`.
+A button named **Julia**
+(with the subtitle "Julia syntax highlighting for Sublime Text 2/3")
+will come up highlighted.
+Click on it, and the package will be installed.
+At this point one should be able to open a Julia source file
+and get it highlighted based upon the syntax of Julia.
+
+[Julia]: https://github.com/JuliaEditorSupport/Julia-sublime
+
+### Install the Terminus and SendCode packages
+
+As before use Package Control to install the terminal emulation package
+[Terminus][],
+and the package to enable communication between a source window
+and the terminal, [SendCode][].
+
+[Terminus]: https://github.com/randy3k/Terminus
+[SendCode]: https://github.com/randy3k/SendCode
+
+## Customization
+
+The key bindings file and the other customization files for the user
+live in the folder for user settings.
+In my case that is
+`C:\Users\PetrKrysl\AppData\Roaming\Sublime Text 3\Packages\User`.
+For brevity I will call this folder `USER`.
+You can find yours through the Command Palette: `Browse Packages`.
```

Comment:
This section should eventually refer to the pages in the Customization section, but they aren't in the state I'd like them to be in just yet. These are general concepts that should only need to be explained once and can then be referred to.